Src/libCZI/subblock_cache.cpp

```diff
@@ -14,7 +14,7 @@ std::shared_ptr<ISubBlockCache> libCZI::CreateSubBlockCache()
 
 ISubBlockCacheStatistics::Statistics SubBlockCache::GetStatistics(std::uint8_t mask) const
 {
-    Statistics result{ 0 };
+    Statistics result;
```

Comment:
I guess zero-initialization was intended here, which would read

`Statistics result{ };`

As it stands, I guess it can be seen 'correct' because the first member 'validityMask' is zero-initialized, which then marks all others as **invalid**. However - with the suggested change, all members are uninitialized, and we could return an uninitialized 'validityMask' then.

Reply:
fixed